dev

```diff
@@ -87,6 +87,23 @@ def main():
 
         return run("yarn desktop", args=args)
 
+    @cmd(
+        cmds,
+        "run-desktop-mainnet",
+        "Run frontend desktop app for development in Mainnet.",
+    )
+    def run_desktop_mainnet(args):
+        run("./scripts/cleanup-desktop.sh")
+        run("yarn install")
+        run("yarn workspace @mintter/ui generate")
+        run("plz build //backend:mintterd //:yarn")
+
+        testnet_var = "MINTTER_P2P_TESTNET_NAME"
+        if testnet_var not in os.environ:
+            os.environ[testnet_var] = "dev"
+
+        return run("MINTTER_P2P_TESTNET_NAME='' yarn desktop", args=args)
```

Comment:
Have you tested that it works as expected? I'm not sure if it will work, doing this prefix for environment variable. I guess we could just clear this environment variable from Python itself, which it confusingly still adds on lines 101-103.

Reply:
You can run this command to see what I'm talking about.

```sh
git apply <<EOF
diff --git a/dev b/dev
index 4aab9043a..0c8fedbe2 100755
--- a/dev
+++ b/dev
@@ -98,11 +98,9 @@ def main():
         run("yarn workspace @mintter/ui generate")
         run("plz build //backend:mintterd //:yarn")
 
-        testnet_var = "MINTTER_P2P_TESTNET_NAME"
-        if testnet_var not in os.environ:
-            os.environ[testnet_var] = "dev"
+        os.environ.pop("MINTTER_P2P_TESTNET_NAME", None)
 
-        return run("MINTTER_P2P_TESTNET_NAME='' yarn desktop", args=args)
+        return run("yarn desktop", args=args)
 
     @cmd(cmds, "build-desktop", "Builds the desktop app for the current platform.")
     def build_desktop(args):

EOF

Reply:
I'm not sure what you mean with this. is there another way you suggest we can have this command running correctly?

Reply:
It did work correctly and running `./dev run-desktop-mainnet` does connect me to the main net @burdiyan

Reply:
I did the change myself and committed.